app/views/users/new.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<% provide :title, t("sign_up") %>
+<h1>Sign up</h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+      <%= f.label :name %>
+      <%= f.text_field :name %>
+      <%= f.label :email %>
+      <%= f.email_field :email %>
+      <%= f.label :password %>
+      <%= f.password_field :password %>
+      <%= f.label :password_confirmation, t("confirmation") %>
+      <%= f.password_field :password_confirmation %>
+      <%= f.submit ("Create my account"), class: "btn btn-primary" %>
+    <% end %>
+  </div>
+</div>
+
+<% if @user.errors.any? %>
```

Comment:
create new file named `_error_messages.html.erb` in  `app/views/shared/` folder
move line 19 -> 32 into the file just is created